.azurepipelines/build-release-packages-1es.yml

```diff
@@ -0,0 +1,359 @@
+    ref: refs/tags/release
+name: $(Build.SourceBranchName)_$(date:yyyyMMdd)$(rev:.r)
+extends:
+  ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/master') }}:
+    template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  ${{ else }}:
+    template: v1/1ES.Unofficial.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: 1ES-PT-Windows-2022
+      os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling-BulkMigrated
+    sdl:
+      sourceAnalysisPool: 1ES-PT-Windows-2022
+    stages:
+    - stage: BuildWindowsAssemblies
+      dependsOn: []
+      jobs:
+      - job: sdkBuildJob
+        displayName: BuildWindowsAssemblies
+        cancelTimeoutInMinutes: 1
+        templateContext:
+          outputs:
+          - output: pipelineArtifact
+            displayName: 'Publish Artifact: framework'
+            path: '$(Build.ArtifactStagingDirectory)'
+            artifactName: windows-assemblies
+        steps:
+        - checkout: self
+          fetchTags: false
+
+        - task: AzureKeyVault@2
+          inputs:
+            azureSubscription: 'AC - Dev Infra & Build Pool'
+            KeyVaultName: 'mobile-center-sdk'
+            SecretsFilter: 'appcenter-sdk-blob-storage-access-secret'
+            RunAsPreJob: false
+
+        - task: PowerShell@2
+          displayName: 'Set Release Version'
+          inputs:
+            targetType: filePath
+            filePath: ./build.ps1
+            arguments: '"version.cake" --Target="SetReleaseVersion"'
+
+        - task: cake-build.cake.cake-build-task.Cake@2
+          displayName: 'Prepare Assemblies'
+          inputs:
+            target: PrepareAssemblies
+            arguments: '--StorageAuthParams="$(appcenter-sdk-blob-storage-access-secret)"'
+            Version: 2.2.0
+
+
+        - task: CopyFiles@2
+          displayName: 'Copy Assemblies'
+          inputs:
+            SourceFolder: '$(Build.SourcesDirectory)\bin'
+            TargetFolder: '$(Build.ArtifactStagingDirectory)'
+
+    - stage: BuildDotNetAssemblies
+      dependsOn: []
+      pool:
+        name: Azure Pipelines
+        image: macos-11
+        os: macOS
+        demands:
+        - msbuild
+      jobs:
+      - job: sdkBuildJob
+        templateContext:
+          outputs:
+          - output: pipelineArtifact
+            displayName: 'Publish Artifact: framework'
+            path: '$(Build.ArtifactStagingDirectory)'
+            artifactName: dotnet-assemblies
+        steps:
+        - checkout: self
+          fetchTags: false
+
+        - task: Bash@3
+          displayName: 'Set Release Version'
+          inputs:
+            targetType: filePath
+            filePath: ./build.sh
+            arguments: '"version.cake"  -t SetReleaseVersion'
+
+        - bash: 'sudo $AGENT_HOMEDIRECTORY/scripts/select-xamarin-sdk.sh 6_12_0'
+          displayName: 'Use Xamarin Bundle 6.12.0'
+
+        - task: JavaToolInstaller@0
```

Comment:
What for do we use Java?

Reply:
This specific version of java was used to build Xamarin projects for Android.